test/test-dev-panel.js

```diff
@@ -86,6 +86,25 @@ exports["test Panel API"] = test(function*(assert) {
   yield closeToolbox();
 
   assert.equal(panel.readyState, "destroyed", "panel is destroyed");
+
+  myTool.destroy();
+});
+
+
```

Comment:
> we should add a comment mentioning that this is important since we use a "chrome" frame, and include a link to this bug too.

I do not understand this. You want me to comment why I destroy a tool here ?